wiki/Installation/en.md

```diff
@@ -21,6 +21,15 @@ This page will tell you how to get osu! working on your Windows device. The [osu
    - If you already have an account, sign in.
    - If you need to register, see [Registration](/wiki/Registration).
 
+## Troubleshooting
+
+**If game installer throws errors when attempting to install, you can try few things:**
+
+- Clean your PC - Scan with anti-virus software, clean the registry. A good program for general clean up is CCleaner. Scan both your computer files and your registry. The installer can fail if there are registry entries from a previous copy of osu! still present on your system.
+- Try reinstalling the latest version of .NET Framework. [You can find it here](https://dotnet.microsoft.com/download).
+
+If you are using a 64-bit Windows 7, you may have difficulties installing osu! correctly. Try to install the 64-bit version of .NET Framework, in case it doesn't come with the osu! installer. [Download it here](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
```

Comment:
```suggestion
If you are using a 64-bit Windows 7, you may have difficulties installing osu! correctly. Try to install the 64-bit version of .NET Framework, in case it doesn't come with the osu! installer. [Download it here](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)

---

**If game installer throws errors when attempting to install, you can try few things:**

- Try reinstalling the latest version of .NET Framework. [You can find it here](https://dotnet.microsoft.com/download).
- Clean your PC - Scan with anti-virus software, clean the registry. A good program for general clean up is CCleaner. Scan both your computer files and your registry. The installer can fail if there are registry entries from a previous copy of osu! still present on your system.
```

Nothing here should have been changed other than the ordering.